src/main/java/dinkplugin/util/Utils.java

```diff
@@ -18,6 +20,8 @@
 @UtilityClass
 public class Utils {
 
+    public static final String WIKI_IMG_URL = "https://oldschool.runescape.wiki/images/";
```

Comment:
Maybe rename to specify it's a URL to be used as a base, not by itself. `WIKI_IMG_URL_BASE`?